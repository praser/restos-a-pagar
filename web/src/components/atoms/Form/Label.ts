import styled from 'styled-components';

const Label = styled.label`
  display: inline-block;
  margin-bottom: 0.5rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5;
  color: #858796;
  text-align: left;
`;

export default Label;
