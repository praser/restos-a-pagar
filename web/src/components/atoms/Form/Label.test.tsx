import React from 'react';

import renderer from 'react-test-renderer';

import Label from './Label';

describe('Label atom component', () => {
  it('is expected to render the component correctly', () => {
    const label = renderer.create(<Label />);
    expect(label).toMatchSnapshot();
  });
});
