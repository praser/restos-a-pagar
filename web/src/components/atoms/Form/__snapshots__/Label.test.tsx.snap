// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label atom component is expected to render the component correctly 1`] = `
<label
  className="sc-bdfBwQ bWfdsa"
/>
`;
